src/tests/multihost/alltests/script/password_expire.sh

```diff
@@ -0,0 +1,12 @@
+expect -f  - <<<'
+    spawn ssh -o StrictHostKeyChecking=no -l ppuser1 localhost
```

Comment:
This script is not expiring password. Please change it to password change or something.
Maybe this is not needed, see comment at L150